front_end/Dockerfile

```diff
@@ -1,21 +1,24 @@
-FROM node:20.10.0
+FROM node:latest
 
-RUN mkdir -p /usr/src/app/frontend
+# expose port
+EXPOSE 3000
```

Comment:
it should be use in .env in case it's NOT in dockerfile since the docker engine will create an image using default environment, the .env file is only used while running the container, meaning that writing :

EXPOSE ${PORT_POSTGRESS} is the same as EXPOSE ""

Unless you export the variable, which defeats the purpose of the .env file.

Reply:
You can pass variables in Dockerfiles using the docker-compose file. It's quite handy.